frontend/src/pages/HomePage.tsx

```diff
@@ -22,28 +23,39 @@ export const HomePage = () => {
   const [loading, setLoading] = useState(false);
   const [open, setOpen] = useState(false);
   const [loginErrorMsg, setLoginErrorMsg] = useState('');
+  const name = accounts[0] && accounts[0].name;
```

Comment:
It seems that this constant isn't used. Can we remove it?

Reply:
got it